src/configure

```diff
@@ -1244,6 +1244,15 @@ elif [ "`uname`" == "Linux" ]; then
       echo MKLLIB = $MKLLIBDIR >> kaldi.mk
     fi
     echo >> kaldi.mk
+
+    echo "MKL_LD_PRELOAD := $MKLROOT/lib/intel64/libmkl_def.so" >> kaldi.mk
```

Comment:
$MKLROOT -> $(MKLROOT)

Reply:
oh, so you are echoing the whole path, ok

Reply:
$MKLROOT, $(MKLROOT) and ${MKLROOT} are both fine in **bash**.

Reply:
mark as resolved.